main.go

```diff
@@ -1,41 +1,141 @@
 package main
 
 import (
+	"errors"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/common/promlog"
 	"github.com/prometheus/common/promlog/flag"
 
+	"github.com/go-kit/kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/version"
 	"gopkg.in/alecthomas/kingpin.v2"
+	"gopkg.in/yaml.v2"
 )
 
 const (
-	namespace                     = "aws_resources_exporter"
-	DEFAULT_TIMEOUT time.Duration = 10 * time.Second
+	namespace                      = "aws_resources_exporter"
+	DEFAULT_TIMEOUT  time.Duration = 30 * time.Second
+	FALLBACK_REGION                = "us-east-1"
```

Comment:
why would we have a fallback region at all? if the list of regions is empty... do nothing?

Reply:
That one is currently only used to retrieve the `all` region, because it always needs a region set.

Depending if we even want to keep that call, I could move that into the function that retrives 'all' regions, but I'd rather get rid of `all` all together and be explicit about the regions using a list.